config-templates/salesforce/opportunities.yaml

```diff
@@ -0,0 +1,117 @@
+data_source:
+  name: salesforce_oppportunities
+  owners:
+    - support@transformdata.io
+  sql_table: salesforce.opportunity ### Update to your opportunities schema.table
+  mutability:
+    type: full_mutation
+    type_params:
+      update_cron: 0 10 * * *
+  identifiers:
+    - name: opportunity_id
+      type: primary
+      expr: id
+    - name: owner_id
+      type: foreign
+      expr: owner_id
+    - name: account_id
+      type: foreign
+  dimensions:
+    - name: ds
+      type: time
+      type_params:
+        is_primary: True
+        time_format: YYYY-MM-DD
+        time_granularity: day
+      expr: created_date::date
```

Comment:
I think `CAST(created_date as DATETIME)` is the most data warehouse agnostic for BQ/Redshift/Snowflake.